app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -163,6 +163,20 @@ class MessageActivity : AppCompatActivity() {
 
         drawableRecyclerAdapter = DrawableAdapter(this, listOfDrawables)
         drawableRecyclerView.adapter = drawableRecyclerAdapter
+
+        val drawableClickListener = object : DrawableAdapter.DrawableClickListener {
+            override fun onClick(position: Int) {
+                drawableRecyclerAdapter.setSelectedItemPosition(position)
```

Comment:
There's no need of this click listener as all operations are done on adapter itself